pages/HistoryPage.tsx

```diff
@@ -0,0 +1,85 @@
+import HeaderButton from "../components/headerButton/headerButton";
+import SubHeader from "../components/subHeader/subHeader";
+import styles from "../styles/HistoryPage.module.css";
+import FooterButton from "../components/footerButton/footerButton";
+import OrderCard from "../components/orderCards/orderCards";
+
+export default function HistoryPage() {
+  return (
+    <main className={styles.main}>
+      <div className={styles.header}>
+        <HeaderButton
+          className={styles.notificationsButton}
+          imgSrc="/bell.svg"
+          onClick={() => alert("Notifications will follow")}
+        />
+        <SubHeader
+          className={styles.subHeader}
+          infoOne="Fahrten"
+          infoTwo="Verdienst"
+          infoThree="Trinkgeld"
+          infoOneNumber={16}
+          infoTwoNumber={[50.5, "€"]}
+          infoThreeNumber={[25.1, "€"]}
+        />
+        <HeaderButton
+          className={styles.moreButton}
+          imgSrc="/more.svg"
+          onClick={() => alert("more functions will follow")}
+        />
+      </div>
+      <div className={styles.cardContainer}>
+        <OrderCard
+          imgSrc="/Card-Placeholder.png"
+          imgSrcStart="/Start.svg"
+          imgSrcEnd="/Target.svg"
```

Comment:
Isn't the start and end Image always the same? If so you could just set it in the component